### PR TITLE
FlowBlockSeal protobuf updates

### LINKED ISSUE #112: BlockSeal type differs from protobuf schema
Related issue: https://github.com/onflow/flow-go-sdk/issues/766

### PR DESCRIPTION
**Pull request by lealobanov**
Closes: #112 

## Description

- Add missing protobuf fields to `FlowBlockSeal` model
- Create `FlowAggregateSignature` model
- Add unit tests for `FlowBlockSeal` class (none existed on this class before)
- Add unit tests for `FlowAggregateSignature` class

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-jvm-sdk/blob/master/CONTRIBUTING.md#styleguides).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels 

## Summary by CodeRabbit

- **New Features**
	- Introduced a new `FlowAggregatedSignature` class for enhanced signature aggregation.
	- Updated `FlowBlockSeal` class with new properties for improved data structure, including `blockId`, `finalState`, `resultId`, and `aggregatedApprovalSigs`.
	- Added methods to retrieve account balances at the latest block and at a specific block height in both the `FlowAccessApi` and `AsyncFlowAccessApi`.

- **Bug Fixes**
	- Ensured proper handling of property names and values in `FlowBlockSeal`.

- **Tests**
	- Added comprehensive unit tests for `FlowAggregatedSignature` and `FlowBlockSeal` to validate functionality and integrity, including equality checks and builder functionality.
	- Enhanced test coverage for account balance retrieval methods, ensuring both success and failure scenarios are adequately tested.

### CODE REVIEW

File: java-example/src/test/java/org/onflow/examples/java/getAccountBalance/GetAccountBalanceAccessAPIConnectorTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+        long balanceAtLatest = balanceAPIConnector.getBalanceAtLatestBlock(address);
+        FlowAccessApi.AccessApiCallResponse<FlowBlock> latestBlockResponse = accessAPI.getLatestBlock(true);
+
+        if (latestBlockResponse instanceof FlowAccessApi.AccessApiCallResponse.Success) {
+            FlowBlock latestBlock = ((FlowAccessApi.AccessApiCallResponse.Success<FlowBlock>) latestBlockResponse).getData();
+            long blockHeight = latestBlock.getHeight();
+
+            // Fetch balance at the same block height
+            long balanceAtHeight = balanceAPIConnector.getBalanceAtBlockHeight(address, blockHeight);
```
[the remark this answers is not in the document]
Comment (lealobanov) [reply java-example/src/test/java/org/onflow/examples/java/getAccountBalance/GetAccountBalanceAccessAPIConnectorTest.java:68]: Will look into this

**Review by franklywatson — APPROVED** [review]
